### PR TITLE
Fix for Base32 Bug

### PR DESCRIPTION
**Pull request by Xor-el**
More Details https://github.com/KvanTTT/BaseNcoding/issues/25

### CODE REVIEW

**Comment by Xor-el** [comment]
Sorry If I sound stupid but could you please explain better or may be show me a sample??
pretty new to this pull request thingy.

**Comment by KvanTTT** [comment]
If you use Visual Studio, click `Edit -> Advanced -> View White Space`. After that you will be able to see all whitespace characters: whitespaces (⋅⋅⋅⋅) and tabs (→). In this project I use the first ones. But you use the second. So, just replace tabs on whitespaces and make me happy :)

**Comment by Xor-el** [comment]
Thanks for the Information.
Done as Requested.

**Comment by Xor-el** [comment]
Done.

File: BaseNcoding.Tests/Base32Tests.cs [hunk 1]
```diff
@@ -12,5 +12,25 @@ public void SetUp()
...
+			Assert.AreEqual(Base32SampleStringResult, encoded);
+		}
+
+		[TestCase(Base32SampleString)]
+		public void Base32CompareEncodeandDecode(string str)
+		{
+			string encoded = Converter.EncodeString(str);
+			string decoded = Converter.DecodeToString(encoded);
+
+			Assert.AreEqual(decoded, encoded);
```
Comment (KvanTTT) [inline BaseNcoding.Tests/Base32Tests.cs]: I guess this is a misprint and you mean the following code?
```CSharp
Assert.AreEqual(str, decoded);
```

File: BaseNcoding.Tests/Base32Tests.cs [hunk 2]
```diff
@@ -12,5 +12,25 @@ public void SetUp()
...
+		[TestCase(Base32SampleString)]
+		public void Base32EncodeCompareToConstantValue(string str)
+		{
+			string encoded = Converter.EncodeString(str);
+
+			Assert.AreEqual(Base32SampleStringResult, encoded);
+		}
+
+		[TestCase(Base32SampleString)]
+		public void Base32CompareEncodeandDecode(string str)
```
Comment (KvanTTT) [inline BaseNcoding.Tests/Base32Tests.cs]: We use CamelCase for all words: Base32CompareEncode**A**ndDecode.

File: BaseNcoding.Tests/Base32Tests.cs [hunk 3]
```diff
@@ -12,5 +12,25 @@ public void SetUp()
 		{
 			Converter = new Base32();
 		}
+
+		public const string Base32SampleString = "Ronald";
+		public const string Base32SampleStringResult = "KJXW4YLMMQ======";
+
+		[TestCase(Base32SampleString)]
+		public void Base32EncodeCompareToConstantValue(string str)
```
Comment (KvanTTT) [inline BaseNcoding.Tests/Base32Tests.cs]: `Base32CompareEncodedAndExpected`

**Comment by Xor-el** [comment]
All Changes Applied as Requested.

**Comment by KvanTTT** [comment]
Not all :) Please replace `Assert.AreEqual(str, encoded);` with `Assert.AreEqual(str, decoded);`.

**Comment by Xor-el** [comment]
Lol, Done.

**Pull request merged**